src/System.Data.SqlClient/tests/ManualTests/System.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -11,6 +11,7 @@
       <Link>Common\System\Collections\DictionaryExtensions.cs</Link>
     </Compile>
     <Compile Include="DataCommon\CheckConnStrSetupFactAttribute.cs" />
+    <Compile Include="SQL\AdapterTest\AdapterTest.cs" />
```

Comment:
You are using TextWriter in the code above. Is there some baseline comparison happening as well? If yes, should a baseline file be included here?